### PR TITLE
AAP-34738: Lightspeed Identify the collections available in the current workspace

### PR DESCRIPTION
**Pull request by goneri**
With this change, the extension builds a list of the local collections of the Workspace
and uses it for the Role Generation wizard.

### CODE REVIEW

**Review by TamiTakamiya — COMMENTED** [review]
It's a general comment on the approach: Can't we just execute `ansible-galaxy collection list` command and parse the command output?

**Comment by goneri** [comment]
> It's a general comment on the approach: Can't we just execute `ansible-galaxy collection list` command and parse the command output?

The right command would now be `ansible-creator init collection foo.bar`, but it won't create the legacy `MANIFEST.json` or broken files. I prefer to avoid the complexity of having to deal with the venv command availability/execution of the command/clean up.

File: src/features/lightspeed/roleGeneration.ts [hunk 1]
```diff
@@ -93,6 +94,20 @@ async function generateRole(
   }
 }
 
+async function getCollectionsFromWorkspace() {
+  const workspaceFolders = workspace.workspaceFolders;
+  let collectionsFound: string[] = [];
+  if (workspaceFolders) {
+    const workspaceDirectories = workspaceFolders.map((f) => f.uri.fsPath);
+    const collectionFinder = new CollectionFinder(workspaceDirectories[0]);
```
Comment (TamiTakamiya) [inline src/features/lightspeed/roleGeneration.ts:102]: I read ["Multi-root workspaces"](https://code.visualstudio.com/docs/editor/workspaces#_multiroot-workspaces) and wondered there might be more than workspaceDirectories.  I think 99.99% of users are not using it wit Ansible extension and do not think we need to support those cases, but we may want to add `console.debug()` when `workspaceDirectories.length > 1`

**Review by TamiTakamiya — APPROVED** [review]
LGTM.  Left one minor comment.

**Review by ssbarnea — DISMISSED** [review]
Please ensure that change is test covered, codecov should pass.

File: src/features/lightspeed/roleGeneration.ts (hunk 1 above)
Comment (goneri) [reply src/features/lightspeed/roleGeneration.ts]: Ah thanks! I forgot about this one. I just adjusted the code to accept a list of workspace directories.

**Pull request merged**